### PR TITLE
Handled problems about installation redis/zookeeper and language set

### PR DESCRIPTION
**Pull request by EminentStar**

### CODE REVIEW

**Comment by EminentStar** [comment]
* Problems
> - Add several commands when installing redis to handle an error while installing zookeeper
> - Create set_language.sh IN ORDER TO set 'language set' SO THAT we could avoid an error when installing pyenv

File: program_install_for_soma.sh [hunk 1]
```diff
@@ -6,18 +6,20 @@ REDIS_VERSION='3.2.8'
 
 if [ ! -d "redis" ]; then
     echo "   3) INSTALLING REDIS-$REDIS_VERSION"
-    wget http://download.redis.io/releases/redis-$REDIS_VERSION.tar.gz
+    wget http://download.redis.io/releases/redis-$REDIS_VERSION.tar.gz -P $REPO_ROOT_PATH
     tar zxvf redis-$REDIS_VERSION.tar.gz
     cd redis-$REDIS_VERSION && make
     mv $REPO_ROOT_PATH/redis-$REDIS_VERSION $REPO_ROOT_PATH/redis
+    cd ..
```
Comment (charsyam) [inline program_install_for_soma.sh]: how about changing this "cd .." to "cd $REPO_ROOT_PATH"
I think it is more robust

File: set_language.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo 'export LANGUAGE=en_US.UTF-8' >> ~/.bashrc
```
Comment (charsyam) [inline set_language.sh]: @EminentStar could you explain more, why happen locale issue?

Comment (EminentStar) [inline set_language.sh]: Right after launching ubuntu instances by vagrant, or AWS(EC2), the locale isn't set in the instances. For this reason, I've experienced failure installing pyenv versions(such as a command: pyenv install 3.5.2). 
I found the failure was of the unset of locale. and I solved the problem after set the locale.

File: program_install_for_soma.sh (hunk 1 above)
Comment (EminentStar) [inline program_install_for_soma.sh]: I agree with you. It's better.

**Comment by charsyam** [comment]
@EminentStar Thank you very much.

**Pull request merged**